sys/mathoid.yaml

```diff
@@ -0,0 +1,86 @@
+info: 
```

Comment:
It isn't used any more (not referenced in the specs), but let me also make sure that it's deleted.